readme.md

```diff
@@ -0,0 +1,130 @@
+
+## Description
+The [NASA's Astrophysics Data System](https://ui.adsabs.harvard.edu/) (NASA/ADS) is a huge database of astronomical and physics papers. Complex literature searches can be performed such that compiling a list of relevant papers for a specific topic can be done easily. This plugin provides an interface to the [NASA/ADS API](http://adsabs.github.io/help/api/) in order to include lists of records in your posts using shortcodes.
+
+With this plugin you are able to
+- easily include simple ADS queries without having to learn the NASA/ADS API.
+- include complex ADS queries by providing the full GET method URL (see the *Shortcodes* section below).
+- customize the format and shown information of the returned list of records at the shortcode level if needed.
+
+Note: versions of this plugin above 0.3 are not backward compatible!
+
+### Shortcodes
+The shortcode `wp_nasaads_query_importer` can be used to query the NASA/ADS and output the returned list of records. It can be inserted into any post with or without providing enclosed content:
+**[wp_nasaads_query_importer *attributes*]**
+or
+**[wp_nasaads_query_importer *attributes*] *format* [/wp_nasaads_query_importer]**
+
+Here, *attributes* are the shortcode's attributes and it is mandatory to either provide
+- at least one of the query attributes described below,
+- the *library* attribute (and combined optionally with query attributes)
+- or the *query* attribute.
+
+In case of enclosed content the *format* is used to format the list of the records. See the *Format* section below.
+
+Please note that for all of the following query attributes the [NASA/ADS search syntax](http://adsabs.github.io/help/search/search-syntax)  is applied, i.e., the shortcode's attribute values are submitted unaltered to the API (with the exception of *author* and *property* as described below). This enables more complex searches, for instance when combined with logical operators like AND, OR, or NOT (read the search syntax for details).
+
+- **author**: search for certain author(s). In case a single author is given the name of the author, i.e., the attribute's value will be submitted with surrounding double quotes (e.g., `Hawking, S.` will be submitted as `"Hawking, S."`).  Technically, a value is considered to be a single author if the string contains no AND, OR, double quotes or parentheses. If you need to include double quotes in the author field, please use single quotes around the attribute's value (see the examples below).
+- **affil**: search for author's affiliations which contain the given string.
+- **journal**: search for articles published in the given journal(s). The acronym of the journal has to be given here (named "bibstem" in the ADS search syntax).
```

Comment:
Please, use the same field name as in ADS (hence, bibstem). This way users only have to learn it once, and it is possible that in the future new fields are created that may collide with the names used here, making everything more confusing.